src/wp-includes/script-loader.php

```diff
@@ -3438,26 +3438,68 @@ function wp_enqueue_command_palette_assets() {
 		'is_network_admin' => is_network_admin(),
 	);
 
+	/**
+	 * Extracts root-level text nodes from HTML string.
+	 *
+	 * @ignore
+	 * @param string $label HTML string to extract text from.
+	 * @return string Extracted text content, trimmed.
+	 */
+
+	$extract_root_text = static function ( $label ) {
```

Comment:
Might as well add PHP type hints:

```suggestion
	$extract_root_text = static function ( string $label ): string {
```